src/main/java/com/engine/GameEngine.java

```diff
@@ -216,6 +216,8 @@ private String getLookResult(String objectToFind) {
 
     public StringBuilder showStatus(String location) {
         StringBuilder builder = new StringBuilder();
+        System.out.println(player.getAreasVisited());
+        System.out.println("Zombies currently following you: " + player.getZombiesFollowing());
```

Comment:
Was this meant to be appended to builder? or were these for testing?

Reply:
Line 220 is for testing as well. I was thinking of incorporating that line to the 'look' command later on

Reply:
sounds good. ill approve